bundles/org.openhab.ui/web/src/components/item/item-mixin.js

```diff
@@ -35,8 +35,8 @@ export default {
      * @returns {string} The error message if the name is invalid, or an empty string if the name is valid.
      */
     validateItemName (name) {
-      if (!/^[A-Za-z0-9_]+$/.test(name)) {
-        return 'Required. A-Z,a-z,0-9,_ only'
+      if (!/^[A-z][A-z0-9_]+$/.test(name)) {
```

Comment:
Should be * iso + in the regex. Single character item names are allowed.

Reply:
Thanks for the catch! Fixed.